recipes/longhorn.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: v1
+kind: kbrew
+app:
+  repository:
+    name: longhorn
+    url: https://charts.longhorn.io
+    type: helm
+  namespace: "longhorn-system"
+  pre_install:
+  - steps:
+    - curl -s https://raw.githubusercontent.com/longhorn/longhorn/v1.1.1/scripts/environment_check.sh | bash
+  post_install:
+  - steps:
+    - echo ${USER}:$(echo ${PASSWORD} | openssl passwd -stdin -apr1) >> auth
+    - kubectl -n longhorn-system create secret generic basic-auth --from-file=auth
```

Comment:
Is this secret is getting used anywhere?

Reply:
Yes, in the Ingress resource